### PR TITLE
net-misc/tigervnc: keywords xinerama dri3

### PR DESCRIPTION
**Pull request by xdch47**
Package-Manager: Portage-2.3.13, Repoman-2.3.3

### CODE REVIEW

**Comment by xdch47** [comment]
Closes: https://bugs.gentoo.org/641708

File: net-misc/tigervnc/tigervnc-1.8.0-r1.ebuild [hunk 1]
```diff
@@ -18,7 +18,7 @@ SRC_URI="https://github.com/TigerVNC/tigervnc/archive/v${PV}.tar.gz -> ${P}.tar.
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sh ~sparc ~x86"
-IUSE="+drm gnutls nls java +opengl pam server systemd +xorgmodule"
+IUSE="+drm gnutls nls java +opengl pam server systemd +xorgmodule xinerama dri3"
 
 CDEPEND="virtual/jpeg:0
 	sys-libs/zlib
```
Comment (SoapGentoo) [inline net-misc/tigervnc/tigervnc-1.8.0-r1.ebuild]: `:=`

File: net-misc/tigervnc/tigervnc-1.8.0-r1.ebuild [hunk 2]
```diff
@@ -153,7 +156,7 @@ src_compile() {
 	if use server; then
 		# deps of the vnc module and the module itself
 		local d subdirs=(
-			fb xfixes Xext dbe $(usex opengl glx "") randr render damageext miext Xi xkb
+			fb xfixes Xext dbe $(usex opengl glx "") $(usex dri3 dri3 "") randr render damageext miext Xi xkb
```
Comment (SoapGentoo) [inline net-misc/tigervnc/tigervnc-1.8.0-r1.ebuild]: `$(usex dri3 dri3 "")` -> `$(usev dri3)`

**Comment by xdch47** [comment]
Thanks for the review ->08bdc46 contains new revision / removed the systemd useflag.

File: net-misc/tigervnc/tigervnc-1.8.0-r2.ebuild [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+	https://dev.gentoo.org/~armin76/dist/tigervnc-1.4.2-patches-0.1.tar.bz2
+	server? ( ftp://ftp.freedesktop.org/pub/xorg/individual/xserver/xorg-server-${XSERVER_VERSION}.tar.bz2	)"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sh ~sparc ~x86"
+IUSE="+drm gnutls nls java +opengl pam server +xorgmodule xinerama dri3"
+
+CDEPEND="virtual/jpeg:0
+	sys-libs/zlib
```
Comment (titanofold) [inline net-misc/tigervnc/tigervnc-1.8.0-r2.ebuild]: As @SoapGentoo pointed out, you need to add := to the end of zlib.

File: net-misc/tigervnc/tigervnc-1.8.0-r2.ebuild [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+
+	if use server; then
+		cd unix/xserver || die
+		eapply "${FILESDIR}/xserver119.patch"
+		eautoreconf
+	fi
+}
+
+src_configure() {
+	use arm || use hppa && append-flags "-fPIC"
```
Comment (titanofold) [inline net-misc/tigervnc/tigervnc-1.8.0-r2.ebuild]: This doesn't mean what you think it means.

Use if statement instead.

Comment (xdch47) [reply net-misc/tigervnc/tigervnc-1.8.0-r2.ebuild]: Statement was taken von the "original" ebuild.
I've to admit, I do not what can go wrong with the logic statement, but I changed it to an if statement.
Out of curiosity: What does it actually mean ?

Comment (titanofold) [reply net-misc/tigervnc/tigervnc-1.8.0-r2.ebuild]: append-flags will only run if `use arm` is false and `use hppa` is true. If `use arm` is true, the rest of the statement is skipped.

**Pull request closed without merging**